cmd/utils/flags.go

```diff
@@ -162,20 +162,24 @@ var (
 	}
 	NetworkIdFlag = cli.Uint64Flag{
 		Name:  "networkid",
-		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby)",
+		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby, 5=Görli)",
 		Value: eth.DefaultConfig.NetworkId,
 	}
-	TestnetFlag = cli.BoolFlag{
+	LegacyTestnetFlag = cli.BoolFlag{ // Deprecated in favor of --goerli, --rinkeby, or --ropsten
```

Comment:
Instead of adding `// Deprecated [...]`, which is redundant with the flag comment and has no special meaning, you could add a `// TODO(q9f): Remove after Ropsten is discontinued`. That leaves a clear mark in the code that something funky is going on here, there's a hint as to what it is and there's also a username whom to contact for more details.